src/crimson/common/perf_counters_service.cc

```diff
@@ -0,0 +1,29 @@
+#include "perf_counters_service.h"
+namespace ceph::common {
+PerfCountersService::PerfCountersService()
+{
+  if (seastar::engine().cpu_id() != 0) {
+    return;
+  }
+  pfCollection.reset(new PerfCountersCollection);
```

Comment:
please refer https://github.com/ceph/ceph/blob/master/CodingStyle for the naming convention. in general, we don't use Hungarian notation in C++ code. `perf_collection` or `perf_coll` might be better alternative.